lib/assessment.js

```diff
@@ -39,7 +59,10 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
-
+  if (!obj){
+    obj = {};
+  } obj[propertyName] = method;
```

Comment:
This line is confusing to read. Unless you're following that curly brace with an `else`, you should start up again on a new line. `grunt nag` will complain at you about things like this; it's great for developing good style habits.